src/components/DataTable/DataTable.js

```diff
@@ -61,11 +61,16 @@ const featureData = ((feature, headers) => {
 
 /**
  * Test if string is a link
- * @param {string} text The text to be tested
+ * @param {Object} text The text to be tested
  * @return {boolean} - if the string is a link or not
  */
-const isLink = text => text.substr(0,7) === 'http://' || text.substr(0,8) === 'https://'
+const isLink = text => text.toString().substr(0,7) === 'http://' || text.toString().substr(0,8) === 'https://'
```

Comment:
What if text is undefined? Or some other type that may not have toString() method?